config/rapidez-checkout-theme.php

```diff
@@ -1,3 +1,25 @@
 <?php
 
-return [];
+return [
+    'account' => [
+        'navigation' => [
+            'orders' => [
+                'icon' => 'heroicon-o-shopping-bag',
+                'href' => '/account/orders',
+                'heading' => 'My orders',
+                'subheading' => 'Place repeat order / View orders',
+            ],
+            'settings' => [
+                'icon' => 'heroicon-o-cog',
+                'href' => '/account/edit',
+                'heading' => 'Account settings',
+                'subheading' => 'Change credentials / Add addresses / Newsletters',
+            ],
+            'logout' => [
+                'icon' => 'heroicon-o-logout',
+                'heading' => 'Logout',
+                'subheading' => 'Logout from your account',
+            ],
+        ],
+    ],
+];
```

Comment:
This comes in handy since we can sync both the main and sidebar navigation with eachother :)